drivers/hmis/app/graphql/types/ac_hmis/esg_funding_service.rb

```diff
@@ -0,0 +1,87 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class AcHmis::EsgFundingService < Types::BaseObject
+    include Types::HmisSchema::HasCustomDataElements
+    description 'AC ESG Funding Service'
+
+    def self.configuration
+      Hmis::Hud::Service.hmis_configuration(version: '2022')
+    end
+
+    field :id, ID, null: false
+    field :client_id, ID, null: false
+    field :client_dob, GraphQL::Types::ISO8601Date, null: true
+    field :mci_ids, [Types::HmisSchema::ExternalIdentifier], null: false
+    field :first_name, String, null: true
+    field :last_name, String, null: true
+    field :project_id, ID, null: false
+    field :project_name, String, null: false
+    field :organization_id, ID, null: false
+    field :organization_name, String, null: false
+    field :date_provided, GraphQL::Types::ISO8601Date, null: false
+    field :fa_amount, Float, null: true
+    field :fa_start_date, GraphQL::Types::ISO8601Date, null: true
+    field :fa_end_date, GraphQL::Types::ISO8601Date, null: true
+
+    custom_data_elements_field
+
+    # Custom data elements are linked to the underlying record (Hmis::Hud::Service or Hmis::Hud::CustomService)
+    # So we pass the record to the resolver.
```

Comment:
This comment isn't really relevant here since we're just working with a CustomService, looks like it was copy-pasted